schemas/input/csl-data.json

```diff
@@ -4,425 +4,477 @@
+          "$ref": "#/definitions/date-variable"
+        },
+        "available-date": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "container": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "event-date": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "issued": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "original-date": {
+          "description": "[Deprecated - Use a date property on the `original` related item instead]",
+          "$ref": "#/definitions/date-variable"
+        },
+        "submitted": {
+          "$ref": "#/definitions/date-variable"
+        },
+        "abstract": {
+          "type": "string"
+        },
+        "annote": {
+          "type": "string"
+        },
+        "archive": {
+          "type": "string"
+        },
+        "archive_collection": {
+          "type": "string"
+        },
+        "archive_location": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "archive-place": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "authority": {
+          "type": "string"
+        },
+        "call-number": {
+          "type": "string"
+        },
+        "chapter-number": {
+          "type": ["string", "number"]
+        },
+        "citation-number": {
+          "type": ["string", "number"]
+        },
+        "citation-label": {
+          "type": "string"
+        },
+        "collection-number": {
+          "type": ["string", "number"]
+        },
+        "collection-title": {
+          "type": "string"
+        },
+        "container-title": {
+          "type": "string"
+        },
+        "container-title-short": {
+          "type": "string"
+        },
+        "dimensions": {
+          "type": "string"
+        },
+        "division": {
+          "type": "string"
+        },
+        "DOI": {
+          "type": "string"
+        },
+        "edition": {
+          "type": ["string", "number"]
+        },
+        "event": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "event-place": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "first-reference-note-number": {
+          "type": ["string", "number"]
+        },
+        "genre": {
+          "type": "string"
+        },
+        "ISAN": {
+          "type": "string"
+        },
+        "ISBN": {
+          "type": "string"
+        },
+        "ISCI": {
+          "type": "string"
+        },
+        "ISMN": {
+          "type": "string"
+        },
+        "ISRC": {
+          "type": "string"
+        },
+        "ISSN": {
+          "type": "string"
+        },
+        "ISWC": {
+          "type": "string"
+        },
+        "issue": {
+          "type": ["string", "number"]
```

Comment:
General comment--perhaps we should make the structure of the JSON and RNC schemas more similar and make definitions for "standard" and "number" and "title" variables, each variable can refer to, rather than have number variables just include both "string" and "number"? This would be a good opportunity since it's a big diff anyway.